### PR TITLE
Completed the about_sets koan

### PR DESCRIPTION
**Pull request by stevendes**
Finished and ready for review!

### CODE REVIEW

File: python2/koans/about_sets.py [hunk 1]
```diff
@@ -11,33 +11,34 @@ def test_sets_make_keep_lists_unique(self):
 
         there_can_only_be_only_one = set(highlanders)
 
-        self.assertEqual(__, there_can_only_be_only_one)
+        self.assertEqual(set(['MacLeod', 'Ramirez', 'MacLeod', 'Matunas',
```
Comment (glpuga) [inline python2/koans/about_sets.py]: Notice that there are repeated elements in the list. A set will ignore those, so the set should be equal to a set with only the unique elements in the table.

**Review by glpuga — CHANGES_REQUESTED** [review]
Good work!.  Just need to fix the first comment.

File: python2/koans/about_sets.py (hunk 1 above)
Comment (stevendes) [reply python2/koans/about_sets.py]: Addressed

**Review by glpuga — APPROVED** [review]
LGTM

**Pull request merged**